subxt/src/tx/tx_progress.rs

```diff
@@ -220,8 +243,16 @@ impl<T: Config, C: OnlineClientT<T>> Stream for TxProgress<T, C> {
 ///    pool about such cases).
 /// 4. `Retracted` transactions might be included in a future block.
 ///
-/// The stream is considered finished only when either the `Finalized` or `FinalityTimeout`
-/// event is triggered. You are however free to unsubscribe from notifications at any point.
+/// Even though these cases can happen, the server-side of the stream is closed, if one of the following is encountered:
+/// - Usurped
+/// - Finalized
+/// - FinalityTimeout
+/// - Invalid
+/// - Dropped
+/// In any of these cases the client side TxProgress stream is also closed.
+/// So there is currently no way for you to tell if an Dropped`/`Invalid`/`Usurped` transaction
+/// reappears in the pool again or not.
+/// You are free to unsubscribe from notifications at any point.
```

Comment:
I think I'd move this to the bottom fo the comment, in part because the rest of this comment is copied from the TxStatus struct in Substrate :)